common/init.sls

```diff
@@ -6,10 +6,24 @@ python2:
       - python
 
 {% if grains['os'] == 'Ubuntu' %}
-python2-dev:
+ubuntu-packages:
```

Comment:
I'd prefer to keep this state ID as just `python2-dev`, and add a new `pkg.installed` state under the `unattended-upgrades` ID to install the respective package. That lets us keep all of the unattended upgrades states together.